torch/distributed/rpc/api.py

```diff
@@ -243,18 +244,18 @@ def rpc_async(to, func, args=None, kwargs=None):
         On worker 0:
         >>> import torch.distributed as dist
```

Comment:
nit, we would need `import torch.distributed.rpc as rpc` here, at least for now

Reply:
Done.